corehq/apps/hqwebapp/crispy.py

```diff
@@ -177,6 +177,8 @@ def edited_classes(context, label_class, field_class):
 class B3MultiField(LayoutObject):
     template = 'hqwebapp/crispy/multi_field.html'
 
+    # Because fields will be passed as-is to the HTML,
+    # care should be taken to ensure they escape user input appropriately
```

Comment:
for these kinds of instances where we assume the arguments are correctly marked as safe upstream, is there a todo to go through these fields to make sure that this is the case? (applies to a few other areas in this PR)

Reply:
I actually think I can drop the comment here, as the only `mark_safe` is for combining all the fields, and those fields themselves are either rendering of templates (which are marked safe themselves, because they're...templates being rendered) or they're the render method of a custom field class, which should also have been individually vetted. Assuming we have examples of the latter, I'm not sure its practical for this effort to go through all of them.

Reply:
I didn't test this, but it seems like maybe `crispy.HTML` is the exception to this? It seems like anything in that tag is implicitly deemed safe, and some of the contents include user input. If my understanding is correct, I'll create another ticket to track these.